src/pages/workspaces/workspace/useWorkspace.test.ts

```diff
@@ -24,10 +23,13 @@ type AjaxContract = ReturnType<AjaxExports['Ajax']>
 
 jest.mock('src/libs/notifications')
 
-jest.mock('src/libs/state', () => ({
-  ...jest.requireActual('src/libs/state'),
-  getUser: jest.fn()
-}))
+type StateExports = typeof import('src/libs/state')
```

Comment:
Cleanup to avoid the ts-ignore at line 127 (forgot to do before previous merge).